WordPress/src/test/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadProgressViewModelTest.kt

```diff
@@ -58,12 +63,24 @@ class BackupDownloadProgressViewModelTest : BaseUnitTest() {
         assertThat(uiStates[0]).isInstanceOf(Content::class.java)
     }
 
+    @Test
+    fun `when started, the progress is set to zero `() = test {
+        val uiStates = initObservers().uiStates
+        whenever(backupDownloadStatusUseCase.getBackupDownloadStatus(anyOrNull(), anyOrNull()))
+                .thenReturn(flow { emit(getStatusProgress) })
```

Comment:
❓: This same line is present in the `setUp()`, do we need it here as well?

Reply:
in 8ae41cc